hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -759,6 +780,11 @@ public PropertyBuilder fromMetaClient(HoodieTableMetaClient metaClient) {
 
     public PropertyBuilder fromProperties(Properties properties) {
       HoodieConfig hoodieConfig = new HoodieConfig(properties);
+
+      for (String key : hoodieConfig.getProps().stringPropertyNames()) {
```

Comment:
might be easier if we go through PERSISTED_CONFIG_LIST instead of all hoodieWriteConfigs

Reply:
you're right. set key-value If the key configured in the `hoodieConfig ` is in `PERSISTED_CONFIG_LIST `.

Reply:
do you mean to say, hoodieConfig.getProps().stringPropertyNames() will be same as PERSISTED_CONFIG_LIST. Just trying to see if we can iterate through smaller list among PERSISTED_CONFIG_LIST and hoodieConfig.getProps().stringPropertyNames()